C-Plus-Plus/other/divisors_of_natural_number.cpp

```diff
@@ -0,0 +1,93 @@
+/* You are given a natural number, for which you have to find out all the possible distinct divisors of the given natural number.
+   The output is always sorted in ascending order.
+   For this we can have various approaches but the most optimized solution has a time complexity of O(sqrt(N)), and a space complexity of O(1).
+*/
```

Comment:
Add a better description. Shift Time and Space Complexity at the end of the Code.